docs/instrumenting-analytics.md

```diff
@@ -1,9 +1,51 @@
 # Instrumenting Analytics
 
-Within javascript inside of dashboard, you can use the function `trackEvent` to trigger a Google Analytics event. For more information, see the [ga.js docs on event tracking](https://developers.google.com/analytics/devguides/collection/gajs/eventTrackerGuide).
+We measure site health and activity using a few different tools:
 
-## Testing your analytics call
+- [Redshift] is where we track KPIs and analyze learning outcomes.
+- [Google Analytics] we use for simple event counting and traffic.
+- [New Relic] is for site health and anything that might trigger an alert.
 
-1. Install the [Google Analytics debugger](https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna)
+As a dev, if you're instrumenting a site health metric, use New Relic.  Otherwise, work with your PM to decide whether to use Redshift or GA.
+
+## Redshift
+
+We send events directly to [Redshift] via [Firehose].  See the wrapper classes for documentation and example code.
+
+- In Ruby use [`FirehoseClient.instance.put_record`][redshift-ruby-docs] ([Example][redshift-ruby-example])
+- In JavaScript use [`firehoseClient.putRecord`][redshift-js-docs]
+
+## Google Analytics
+
+In JavaScript, you can use the function [`trackEvent`][ga-js-code] ([Example][ga-js-example]) to report an event to [Google Analytics]. For more information, see the [ga.js docs on event tracking][ga-js-docs].
+
+### Testing your Google Analytics call
+
+1. Install the [Google Analytics debugger]
 2. Test locally
 3. Look for the analytics call in the Chrome debugger
+
+## New Relic
+
+[New Relic] has separate APIs for recording "events" and "metrics."  It's a little hard to find good resources on the distinction; [this thread][new-relic-events-vs-metrics] may be helpful.
+
+- In Ruby, use [`NewRelic::Agent.record_custom_event`] and [`NewRelic::Agent.record_metric`] ([Example][new-relic-ruby-example])
+- In JavaScript, we record custom events with [`logToCloud.addPageAction`][new-relic-js-docs] ([Example][new-relic-js-example]).  We haven't set up any custom metrics from JavaScript yet.
+
+[Firehose]: https://aws.amazon.com/kinesis/firehose/
```

Comment:
@tanyaparker You might want to share this cool technique for putting all the links down the bottom with content editors.